### DISCUSSION TITLE
option createChildContextWithAs is false in default

### DISCUSSION THREAD

**Discussion by Auge19** [knockout/tko#191] [Q&A] [answered]
Hi!

our team migrated from ko to tko beta and got some issues in this progress. One was a "cannot bind to $parent" (don't remember to exact message right now). After some missleading error messages and debugging-time, we learned of the existence of the createChildContextWithAs option. That causes out problem, because the default is false and in the old behaviour it would be true.

So why is the default false? Totally fine, when it's for Performance! But was it a design decision?
Next question: Is there a more detailed migration guide. The readme and homepage seems quite outdated at some parts...

**Comment by phillipc** [comment, marked as answer]
The current default allows some tko features, see some tests like:

- Should provide access to observable items (foreachBehaviors.ts)
- Should not re-render the nodes when an observable value changes (withBehaviors.ts)
- Should remove nodes with an observable value become falsy (withBehaviors.ts)

I would leave the default as it is, but a better readme is the way to go!
